jrodriguezdiazz/jrodriguezdiazz-portfolio
# Add pages for 404 and 500 errors

## Description

The corresponding pages have been added for routes that do not exist (404) and if something fails (500)




---

## Motivation and Context

These pages have been added to provide a better user experience in case you try to access a page that does not exist or something explodes





---

## How Has This Been Tested?

1. Go to: https://jrodriguezdiazz-portfolio-6i1qmtwin-jrodriguezdiazz.vercel.app/
2. Try to access any route other than the following:
   1. `about`
   2. `contact`
   3. `experience`
   4. `projects`
3. You should see a page like the ones shown below





---

## Screenshots (if appropriate):
| Desktop      | Mobile |
| ----------- | ----------- |
| ![image](https://user-images.githubusercontent.com/24766287/141650272-a2566518-7ba2-421d-8277-86a35faac477.png) | ![image](https://user-images.githubusercontent.com/24766287/141650317-1ce861bb-e4a7-4fab-bacf-608320cd00dd.png) |



---

## Types of Changes




---

## Checklist:





---

# Other Notes
